### DISCUSSION TITLE
500

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#184] [Comments]
# 500

설치 요구사항 Jenkins를 설치하고 실행하기 위해서는 아래의 조건을 만족해야 한다. 호스트 머신이 램 최소 256 MB (권장 2 GB) 젠킨스 및 도커 이미지를 위한 여분의 저장공간 10 GB 아래의 소프트웨어 들이 설치 되어 있어야함 Java 11, 17, or 21 Docker (아직 설치가 안되어있다면 Get Docker 참고) 다운로드 https://www.jenkins.io/download/ 링크에서 젠킨스를 다운 받는다. OS에 맞춰 원하는 무엇으로 다운받아도 상관 없으나 본 튜토리얼은 .war 파일로 진행한다 실행 다운로드 받은 war 파일을 java로 실행한다. httpPort는 원하는 값을 넣으면 되는데, 8000으로 진행해보도록 한다. java -jar jenkins.war --h..

https://shanepark.tistory.com/500

**Comment by answndud** [comment]
무슨 브라우저 theme 쓰시나요? 저도 적용하고 싶습니다!!

**Reply by ShanePark** [reply]
안녕하세요.   
우분투에서 firefox 에 [WhiteSur-gtk-theme](https://github.com/vinceliuice/WhiteSur-gtk-theme/tree/master/src/other/firefox) 테마의 Monterey 입혀서 쓰고 있습니다.

**Reply by ShanePark** [reply]
다시 보니 웹브라우저 스샷 있는건 맥북에서 그냥 크롬 켠걸로 보이는데, 특별히 테마는 없고 그냥 다크모드설정만 되있는 거에요

**Reply by answndud** [reply]
답변 감사합니다!
